### ISSUE TITLE
Opening KMZ files

### ISSUE THREAD

**Issue by baarkerlounger** [nextcloud/maps#218]
Probably a feature request (or maybe there's a way to do it already) but it would be great if Nextcloud Maps could open `.kmz` files exported from Google My Maps.

**Comment by jancborchardt** [comment]
Yep! @baarkerlounger are you interested in looking into how to do that? :) Even just finding links to libraries or how others do it is a good start!

**Comment by v1r0x** [comment]
`.kmz` are only zipped `.kml` files. Unzipping them and doing the kml stuff should be enough :wink:

**Comment by idevwebs** [comment]
How do we open kmz/kml? I tried the import favorites and nav to either type and just get a failed to import message. 

https://github.com/windycom/leaflet-kml

**Comment by Albirew** [comment]
before i open a new issue for nothing, kmz import on maps seems to work great (except when trying to import kml/kmz with non-latin characters), so is there another issue that keeps this issue open?

**Comment by Batcastle** [comment]
I'm very interested in getting this working as I have a map in my Google Drive I want to move out and into my Nextcloud. What needs to be done to get this working?

**Comment by Abijeet** [comment]
I think a feature to import KML files would be quite useful to migrate away from Google Maps.

I used Google Takeout to export some saved places that I have on Google Maps. I then used this script to https://github.com/akvadrako/json2kml to generate a KML from the JSON files that were exported. Now I do not see a way to import the KML file.
